pyproject.toml

```diff
@@ -6,14 +6,15 @@ readme = "README.md"
 authors = [{name = "Sym Roe"}, {name = "Virginia Dooley"}]
 
 dependencies = [
-	"django>=3.2,<4.3",
-	"django-pipeline==3.0.0",
-	"markdown",
-	"django-localflavor==4.0",
-	"setuptools==69.2.0",
-	"whitenoise",
-	"pysass",
-	"jsmin<3.1",
+    "django>=3.2,<4.3",
+    "django-pipeline==3.0.0",
+    "markdown",
+    "django-localflavor==4.0",
```

Comment:
Any particular reason we are pinned to exact patch versions of `django-pipeline` and `django-localflavor` here, but no constraint at all on all the others?

Reply:
I don't know, I suspect they were the latest versions at the time. We should be quite flexible with these dependencies I think, so as not to constrain ourselves too much in other projects

Reply:
widened these in https://github.com/DemocracyClub/dc_django_utils/pull/88/commits/a044e4c18db28198b3c1d19740ff85cec98cee6b